requirements.txt

```diff
@@ -28,31 +28,31 @@ httplib2==0.11.3
 idna-ssl==1.1.0
 idna==2.7
 imageio==2.4.1
-inflect==1.0.0
+inflect==1.0.1
 jaraco.itertools==2.5.2
 lxml==4.2.5
 mechanicalsoup==0.11.0
 more-itertools==4.3.0
 moviepy==0.2.3.5
 multidict==4.4.2
-numpy==1.15.1
+numpy==1.15.2
 oauthlib==2.1.0
-pillow==5.2.0
-protobuf==3.1.0.post1
+pillow==5.3.0
+protobuf==3.6.1
 psutil==5.4.7
 pushbullet.py==0.11.0
 pyasn1-modules==0.2.2
 pyasn1==0.4.4
 pyslack==0.4.0
 python-dateutil==2.7.3
 python-magic==0.4.15
-readlike==0.1.2
+readlike==0.1.3
 reparser==1.4.3
 requests-oauthlib==1.0.0
 requests==2.19.1
 rsa==4.0
-selenium==3.14.0
-simplejson==3.16.0
+selenium==3.14.1
+simplejson==3.16.1
```

Comment:
The pyup bot will keep tying to update `simplejson` to `3.16.1` which got unpublished.